include/slang/ast/Lookup.h

```diff
@@ -83,6 +83,9 @@ enum class SLANG_EXPORT LookupFlags {
     /// the $unit compilation unit scope.
     DisallowUnitReferences = 1 << 14,
 
+    /// Allow unnamed generate blocks to be looked up by their external name.
+    AllowUnnamedGenerate = 1 << 15,
+
```

Comment:
You need to update the SLANG_BITMASK macro below to reference this since it is now the highest valued flag. Also this should be manually added to the Python bindings in SymbolBindings.cpp